scope3_methodology/api/api.py

```diff
@@ -89,6 +92,18 @@ async def startup_event():
     property_defaults[PropertyChannel.STREAMING] = Property.load_default_yaml(
         "generic", property_defaults_file_path, PropertyChannel.STREAMING.value
     )
+    end_user_device_defaults[EndUserDevices.PERSONAL_COMPUTER] = EndUserDevice.load_default_yaml(
```

Comment:
define list of end user devices that we need to pull in and then iterate over the list pulling them in and populating the dict? rather than repeating same thing N times per device type

Reply:
or, rather than define list, just iterate over the EndUserDevices enum

Reply:
I have a follow up PR that will be up once this merges to address https://github.com/scope3data/methodology/issues/66 Following existing pattern right now just for consistency before cleanup.